### ISSUE TITLE
TextField doesn't work with both textarea and outlined set

### ISSUE THREAD

**Issue by nworden** [material-components/material-components-web-react#559]
Steps to repro:
1. Import TextField and Input from @material/react-text-field
2. Include the following code:
```javascript
<TextField label="Giraffe Textarea" outlined textarea>
  <Input value="giraffe giraffe giraffe" inputType="textarea" />
</TextField>
```

Expected: something like a regular outlined text field, but with a textarea.
Actual: The field has two outlines (one of which doesn't get notched) and the label doesn't float:
![materialreact-outlinedtextarea](https://user-images.githubusercontent.com/9059304/50310053-e405a680-046e-11e9-9e5c-02e8a84b0c25.png)

Regular-sized outlined TextFields work for me, so I guess the problem is probably with the textarea part or the combination of the two (I'm not sure what non-outlined textareas are supposed to look like).

I tried the Material web components example code (copied at the end of this post) for an outlined textarea and it worked fine, and it looks like the HTML produced by the React components is different in a number of ways (the example from the components repo doesn't use a mdc-text-field--outlined class, and the notch seems to be implemented totally differently).

```html
<link rel="stylesheet" href="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.css">

<div class="mdc-text-field mdc-text-field--textarea">
  <textarea id="textarea" class="mdc-text-field__input" rows="8" cols="40"></textarea>
  <div class="mdc-notched-outline">
    <div class="mdc-notched-outline__leading"></div>
    <div class="mdc-notched-outline__notch">
      <label for="textarea" class="mdc-floating-label">Textarea Label</label>
    </div>
    <div class="mdc-notched-outline__trailing"></div>
  </div>
</div>

<script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
<script>
  mdc.textField.MDCTextField.attachTo(document.querySelector('.mdc-text-field'));
</script>
```

**Comment by moog16** [comment]
@nworden I believe this didn't worked before, so this is a new issue. The notched-outline has been redesigned hence the new markup. Please see the [new notched outline](https://github.com/material-components/material-components-web/commit/9741233).

**Comment by moog16** [comment]
This is fixed in #697. Please see screenshots working https://storage.cloud.google.com/screenshot-uploads/text-field/textArea/dde78e3f154a8b910a989f8ce96e320e7ad2b3e199e6e7a81034174c598cbd9d.golden.png?organizationId=433637338589&_ga=2.177225829.-1123561096.1535414695

**Comment by nworden** [comment]
Thanks Matt!
